mongodb/Module.md

```diff
@@ -54,7 +44,7 @@ database = "<DATABASE_NAME>"
 collection = "<COLLECTION_NAME>"
 ```
 
-### Step 3: Initialize the Mongodb Client giving necessary credentials
+#### Step 3: Initialize the Mongodb Client giving necessary credentials
 
 You can now enter the credentials in the mongo client config. If you use this client for a particular database then you can pass the database name along with config during client initialization(It is optional). Otherwise you can pass the database name for each remote method call. This is not recommended unless you need to connect more than one database using a client. You need to set the database using atleast one of these methods.
```

Comment:
We need to add specific instructions here. 
1. Where do we find the client config? 
2. If you use this client for a particular database then you can pass the database name along with config during client initialization(It is optional) - is that what we are doing here?
3.  pass the database name for each remote method call - where do we show how to do this?

Reply:
Fixed by 89976637b8fed8c9a16842370da109f71b035569.